generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -263,6 +269,9 @@ public void updateUser(String firstName, String lastName, String email, String l
                 <%_ if (databaseType == 'mongodb' || databaseType == 'cassandra') { _%>
                 userRepository.save(user);
                 <%_ } _%>
+                <%_ if (searchEngine == 'elasticsearch') { _%>
+                // userSearchRepository.save(user); // left out by design
```

Comment:
I dont see why this is required here. can you please rebase this branch with latest master and then remove this?